samples/cpp/benchmark_app/main.cpp

```diff
@@ -1126,6 +1132,9 @@ int main(int argc, char* argv[]) {
                 if (FLAGS_pc) {
                     slog::info << "Performance counts for " << ireq << "-th infer request:" << slog::endl;
                     printPerformanceCounts(reqPerfCounts, std::cout, getFullDeviceName(core, FLAGS_d), false);
+                } else if (FLAGS_pcsort == pcSort || FLAGS_pcsort == pcNoSort || FLAGS_pcsort == pcSimpleSort) {
```

Comment:
`-pc` and -`pcsort` should be mutually exclusive or `--pcsort` should have first priority and checked first.

Reply:
I think we can extent of the two parameters "-pc" and "-pcsort", add the "proportion" as a default information then print the performance count, and change the "-pc" as a string parameters with optional parameter "sort","sort_simple"